.github/ubiquibot-config.yml

```diff
@@ -29,7 +29,7 @@ priority-labels:
   - name: "Priority: 4 (Emergency)"
     weight: 5
 auto-pay-mode: true
-analytics-mode: true
+disable-analytics: true
```

Comment:
```suggestion
```

This should be enabled by default, right?

Reply:
[ref](https://github.com/ubiquity/ubiquibot/pull/512#pullrequestreview-1546975867)

Reply:
Okay if the value is `undefined` then the bot should enable analytics. 

I'm pretty sure in the specification I wrote that the value should be hidden from the default configuration. 

However analytics should always be on UNLESS we deliberately add the property to disable it.

Reply:
ohhh